src/main/cli/cli.c

```diff
@@ -1316,14 +1316,16 @@ static void cliSerial(const char *cmdName, char *cmdline)
     uint8_t validArgumentCount = 0;
 
     const char *ptr = cmdline;
-
-    int val = atoi(ptr++);
+    char *tok = cmdline;
+    int val = findSerialPortByName(strsep(&tok, " "), strcasecmp);
```

Comment:
```suggestion
    serialPortIdentifier_e val = findSerialPortByName(strsep(&tok, " "), NULL);
    if (val == SERIAL_PORT_NONE) {
```
if UART name failed, fallback to numeric parsing, with 0->UART. This way saved dump with number will parse correctly

Reply:
val is used elsewhere later in the function. have added explicit variable for identifier.